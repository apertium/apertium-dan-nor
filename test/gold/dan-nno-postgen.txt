[Hac3cvhrs00E]
når det gjeld størstedelen [/option]
[/Hac3cvhrs00E]
