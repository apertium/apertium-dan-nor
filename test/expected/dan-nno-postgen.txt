[06Jrtnx4CU6Z#0] Etterkvart som der er komme meir styr på bruken av miljøgifter generelt, men især i landbruket, blir sett vandrefalkar òg i Danmark i eit stadig større mengd på trekk mot Noreg og Sverige.
[/06Jrtnx4CU6Z]
[13lkKfMMjli-#0] Vandrefalkens føde består av mellomstore fuglar som han fangar i lufta.
[/13lkKfMMjli-]
[1IulJoMbI3rr#0] Her blir hevda han å kunne nå ein fart på 390 km\/t.
[/1IulJoMbI3rr]
[1XR8ugqC8dyr#0] Hun-vandrefalken som er størst, klarer at ta bytte på heilt opp til 1 kg.
[/1XR8ugqC8dyr]
[1w4SjZ34bVt1#0] Vandrefalken var gjennom 1960-åra og 1970-åra sterkt i tilbakegang i Skandinavia og mange andre stader.
[/1w4SjZ34bVt1]
[2iUkU1dA7rtZ#0] Det dreier seg ofte om *duer, vadefuglar, mindre ender, kråkefuglar og trostar.
[/2iUkU1dA7rtZ]
[5x8TEJ_RVxFV#0] Den 30. juni vart tre ungar (to hannar og ein ho) ringmerket nær Hammarhus på Bornholm.
[/5x8TEJ_RVxFV]
[6jYihw37Zx5J#0] meir verdifull
[/6jYihw37Zx5J]
[7Iu3xZgFBVRz#0] Vandrefalken gjorde det siste yngleforsøket sitt i Danmark i 1972, men er no tilbake som dansk ynglefugl, ettersom han sidan 2002 har yngla på Møns *klint (2002: 2 ungar, 2003: 3 ungar).
[/7Iu3xZgFBVRz]
[8jRQMBgLNpMl#0] mest borgarlege
[/8jRQMBgLNpMl]
[9qPDzahWsJRS#0] Vandrefalk frå England.
[/9qPDzahWsJRS]
[AGf_CIcG6bLB#0] Den er kjent for å vera blant verdas hurtigaste fuglar, når den styrtdykkaren mot eit bytte.
[/AGf_CIcG6bLB]
[CGIBBF-qJiQP#0] Vandrefalk (*Falco *peregrinus) er ein rovfugl som er utbreidd over alle kontinent på nær Antarktis.
[/CGIBBF-qJiQP]
[GuCJGMSC55jy#0] I 2009 vart der for første gang sidan 1937 (der tre ungar vart ringmerkt på Bornholm) ringmerket *vandrefalkeunger i Danmark.
[/GuCJGMSC55jy]
[Hac3cvhrs00E#0] når det gjeld størstedelen
[/Hac3cvhrs00E]
[ILH_TKWtSLEy#0] Den 18. mai vart der ringmerkt tre ungar på Stevns Klint (to hannar og ein ho) og den 19. mai fire ungar på Møns Klint (to hannar og to *hunner).
[/ILH_TKWtSLEy]
[Iy3D9K4sfVe8#0] Der var seinst i 2012 12 ynglepar i Danmark, korav 6 par hadde suksess med å få ungar på *vingerne.
[/Iy3D9K4sfVe8]
[JXsiSKP2HO9_#0] I 2004 prøvde ein nytt dansk *vandrefalkepar å yngla på Bornholm, men det mislykkast di verre pga. manglande vern av yngleområdet.
[/JXsiSKP2HO9_]
[MvI9A0AN1aXS#0] Vandrefalken har sidan 2007 yngla med suksess på Stevns Klint (2007: 2 ungar, 2008: 3 ungar) og han har òg etablert reie i 2009.
[/MvI9A0AN1aXS]
[UDEmh40X_Na9#0] av
[/UDEmh40X_Na9]
[WGUkSKGKhcJ3#0] Især insekticidet DDT SOM medførte *tyndskallede egg, påverka vandrefalkens *reproduktionsevne.
[/WGUkSKGKhcJ3]
[X2KfDHUw343q#0] mest borgarleg
[/X2KfDHUw343q]
[XIV_kHEp8XGA#0] Ynglebestanden er vurdert som sårbar på den danske *rødliste.
[/XIV_kHEp8XGA]
[XZnUbh3Z2JLN#0] Miljøgifter var den store synder då dei øydela fuglanes *frugtbarhed.
[/XZnUbh3Z2JLN]
[XrLP3qOkI-cd#0] Ynglefugl i Danmark.
[/XrLP3qOkI-cd]
[YB3NJdSjwuh8#0] Miljøgifter.
[/YB3NJdSjwuh8]
[YiITmoSuWXBn#0] Dette aerodynamiske designet er òg vorte brukt av flykonstruktørar for å auka kampflys fart.
[/YiITmoSuWXBn]
[e_tufqM_NiFZ#0] Det finst omkring 16 underartar.
[/e_tufqM_NiFZ]
[gJh2RMTY1F_N#0] Wikipedia er ein internasjonal internettbasert encyklopedi som blir utgitt av den ideelle organisasjonen Wikimedia Foundation
[/gJh2RMTY1F_N]
[gchkSnxi0muM#0] Fuglen dannar med sine *smalle *vinger ein pilforma profil når han flyr.
[/gchkSnxi0muM]
[ro_zQTI_DdSW#0] Zoologen Rachel Carson påpeika nett dette problemet allereie i 1962 i boka si *Silent Spring.
[/ro_zQTI_DdSW]
[yCVw30jwzGCn#0] Den er ein av dei føretrekte fuglane blant falkonerar.
[/yCVw30jwzGCn]
